examples/AccountManagement/GetAccountHierarchy.cs

```diff
@@ -0,0 +1,233 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Ads.GoogleAds.V3.Services;
+using Google.Api.Gax;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// This example gets the account hierarchy of the specified manager account. If you don't specify
```

Comment:
The code is LGTM, could you please format it to 100 lines length? 

Also, mark https://github.com/googleads/google-ads-dotnet/issues/24 as fixed once this PR is merged.

Reply:
Fixed.